### ISSUE TITLE
Missing LZipArchive class

### ISSUE THREAD

**Issue by sbruyere** [adamhathcock/sharpcompress#582]
It appears on the "supported formats" but I canno't find this class available.

**Comment by adamhathcock** [comment]
You're right.  I guess I thought I had those classes but I don't.  LZ files are currently only usable in connection with Tar it seems.

It would be easy to add them as they're as simple as GZip.  

I removed the mentions from the format MD for now.

**Comment by sbruyere** [comment]
Thank you for this clarification! It was just in case you had simply forgotten to make this class available. :)
